test/sortitionPoolTest.js

```diff
@@ -2,6 +2,19 @@ const chai = require("chai")
 const expect = chai.expect
 const { ethers, helpers } = require("hardhat")
 
+async function withdrawRewards(
+  pool,
+  owner,
+  operatorAddress,
+  beneficiaryAddress,
+) {
+  const tx = await pool
+    .connect(owner)
+    .withdrawRewards(operatorAddress, beneficiaryAddress)
+  const withdrawLogs = await tx.wait()
+  return parseInt(withdrawLogs.logs[0].data)
+}
```

Comment:
We can also do it this way:
```
    async function withdrawRewards(
      pool,
      owner,
      operatorAddress,
      beneficiaryAddress,
    ) {
      const amount = await pool
        .connect(owner)
        .callStatic.withdrawRewards(operatorAddress, beneficiaryAddress)
      await pool.connect(owner).withdrawRewards(operatorAddress, beneficiaryAddress)
      return amount
    }
```

Reply:
[b8fd7c8](https://github.com/keep-network/sortition-pools/pull/166/commits/b8fd7c821b3af00ea7e608950f9475f658e6dcc6)